torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3593,6 +3593,27 @@ def _maybe_evaluate_static(
         symbols = list(expr.free_symbols)
 
         # Apply known runtime asserts
+        guards_exprs = []
+        for g in self.guards:
+            e = self.simplify(g.expr)
```

Comment:
On master, we're not calling simplify yet. I guess it's harmless enough to do here though.